main.go

```diff
@@ -66,5 +82,5 @@ func main() {
 		r.POST("/receive", receiveHandler(producer, serializer))
 	}
 
-	r.Run()
+	log.Fatal(r.Run())
```

Comment:
@mingqing I think this log.Fatal is not right.

Reply:
If running in GIN_MODE="release", it is more convenient for troubleshooting if an exception occurs before startup, like log: (I think it's better to add this, especially deployment on k8s)

`{"level":"fatal","msg":"listen tcp :10086: bind: address already in use","time":"2020-06-21T20:28:23+08:00"}`